Tests/MapboxMapsTests/Style/ExpressionTests/ExpressionTests.swift

```diff
@@ -97,4 +99,45 @@ final class ExpressionTests: XCTestCase {
 
         XCTAssertEqual(validArg, argument)
     }
+
+    func testGeoJSONObjectExpression() throws {
+        let coloradoCorners: [CLLocationCoordinate2D] = [
+            .init(latitude: 37, longitude: -109-2/60-48/60/60),
+            .init(latitude: 37, longitude: -102-2/60-48/60/60),
+            .init(latitude: 41, longitude: -102-2/60-48/60/60),
+            .init(latitude: 41, longitude: -109-2/60-48/60/60),
+            .init(latitude: 37, longitude: -109-2/60-48/60/60),
+        ]
+        var colorado = Feature(geometry: .polygon(Polygon([coloradoCorners])))
+        colorado.identifier = "CO"
+        colorado.properties = [
+            "population": 5_773_714,
+        ]
+
+        let withinExpression = Exp(.within) {
+            GeoJSONObject.feature(colorado)
+        }
+        XCTAssertNotNil(withinExpression.arguments.first)
+        guard let withinObject = withinExpression.arguments.first else { return }
```

Comment:
```suggestion
        let withinObject = try XCTUnwrap(withinExpression.arguments.first)
```

Reply:
Also, I think I'd rename the local variable to `firstArgument`. The fact that we're dealing with a "within" expression is clear enough from the context.